src/Accelerators/NNPA/Conversion/ONNXToZHigh/RewriteONNXForZHigh.td

```diff
@@ -95,17 +95,6 @@ def replaceONNXBatchNormalizationInferenceModePattern : Pattern<
 //
 //===----------------------------------------------------------------------===//
 
-
```

Comment:
migrated the code elsewhere so that it can be reused, as it was needed to support the reshape op.